std/random.d

```diff
@@ -1298,20 +1340,33 @@ random number sequences every run.
 
 Returns:
 A single unsigned integer seed value, different on each successive call
+Note:
+In general periodically 'reseeding' a PRNG does not improve its quality
+and in some cases may harm it. For an extreme example the Mersenne
+Twister has `2 ^^ 19937 - 1` distinct states but after `seed(uint)` is
+called it can only be in one of `2 ^^ 32` distinct states regardless of
+how excellent the source of entropy is.
```

Comment:
This is a great piece of documentation, which I would suggest extending with discussion on how to more effectively seed generators (for example, `MersenneTwisterEngine` exposes a method to seed the generator using an `InputRange` of random words).

However, I would suggest not coupling it with these changes (it's independent of them) and submitting it in a separate PR.

Reply:
While of course, separating PR has advantages, they also come with an additional overhead of creating and reviewing them.
So it's really ok to combine doc updates of the function currently touched in a PR. No need for the extra work.